packages/vm/src/runBlock.ts

```diff
@@ -971,3 +986,48 @@ export const accumulateRequests = async (_vm: VM): Promise<CLRequest[]> => {
   }
   return requests
 }
+
+const _accumulateEIP7002Requests = async (vm: VM, requests: CLRequest[]): Promise<void> => {
+  // TODO PERFORM LOGIC TO CHECK IF CONTRACT EXISTS
```

Comment:
Is this todo still valid?  Doesn't your code below check if the contract exists?

Reply:
Yup, I forgot to remove the TODO

Reply:
I removed it, 6dbb5f2990bc1b96fcfe89aeb78c579b77ad67ec